featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -606,7 +608,9 @@ def _build_agg_features(self, all_features,
 
                     self._handle_new_feature(new_f, all_features)
 
-    def _features_by_type(self, all_features, entity, variable_type, max_depth):
+    def _features_by_type(
```

Comment:
to keep the style of our other code - let's keep this as one line or break up the arguments over two lines